ui/job-view/details/BugFiler.jsx

```diff
@@ -520,7 +523,7 @@ export class BugFilerClass extends React.Component {
                     this.setState({ productSearch: evt.target.value })
                   }
                   type="text"
-                  placeholder="Firefox"
+                  placeholder="e.g. Firefox, Toolkit, Testing"
```

Comment:
Having this just read "Firefox" kept throwing me off, making me thing the text input box had been prefilled as the "Firefox" product (or maybe it looks like a <select> element preset to "Firefox". This should remove some confusion, even if not related to the rest of this PR.